client/components/button-fill.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <component
+    class="button-fill"
+    :href="href"
+    :is="tag"
+    :to="to"
+    @click="onClick"
+  >
+    {{ label }}
+  </component>
+</template>
+
+<script>
+export default {
+  name: 'button-fill',
+  props: {
+    href: {
+      type: String,
+    },
+    label: {
+      type: String,
+    },
+    tag: {
+      type: String,
+      default: 'button',
+    },
+    to: {
+      type: Object,
+    },
+  },
+  methods: {
+    onClick(...args) {
+      this.$emit('click', ...args);
+    },
+  },
+};
+</script>
+
+<style lang="stylus">
+.button-fill {
+  cursor: pointer;
+  display: inline-block;
+  padding: 13px 21px;
```

Comment:
Do you have any spacing and/or padding variables that are shared throughout the app? (Similar to Base Web's $theme.spacing)

Reply:
Right now we don't but I think that would be a good idea for a future PR